project-reports/2023/2023-Q3-Hyperledger-FireFly.md

```diff
@@ -0,0 +1,124 @@
+
+Hyperledger published the case study [The Hyperledger FireFly Story: Kaleido Taps into the Hyperledger Community to Develop Next-Gen Solutions](https://media.licdn.com/dms/document/media/D561FAQFHEjkdNUvJEg/feedshare-document-pdf-analyzed/0/1692713935821?e=1694044800&v=beta&t=KLpydg2_L7VbFMuSJpm3OsnKUJbxw3Q6dDzto_UPFGE).
+
+We have onboarded 3 new maintainers to the project and one maintainer has transitioned to a new company. These new maintainers include Matthew Whitehead & Chengxuan Xing from Kaleido and Dzianis Andreyenka from Instincttools. Vinod Damle has moved to Fidelity Investments and will continue as a maintainer.
+
+The Hyperledger FireFly team has been working closely with developers at OneOf over the past year as they built a new FireFly blockchain connector for the Tezos blockchain. They also demod this new blockchain connector at a recent FireFly Community Call. This codebase is in the process of being open sourced and donated to the Hyperledger FireFly project.
+
+Community adoption updates:
+
+- The DFCRC and Reserve Bank of Australia released the [results of their CBDC Pilot](https://www.rba.gov.au/payments-and-infrastructure/central-bank-digital-currency/pdf/australian-cbdc-pilot-for-digital-finance-innovation-project-report.pdf) which leveraged Hyperledger FireFly. Some use cases tested in the pilot include:
+  - A wholesale car dealer generated a tokenized invoice representing a claim on a wholesale buyer.
+  - Offline customer-to-merchant transactions using NFC-enabled smart cards and pilot CBDC between RMIT University and Southern Cross University
+  - Two banks trading tokenized debt securities in wholesale markets
+
+Community members have given talks featuring Hyperledger FireFly at a number of events including:
+
+- Consensus 2023: [The CBDC Tech Stack: How One Ethereum Firm is Working with Central Banks](https://www.youtube.com/watch?v=L1MeqCRfQDw&ab_channel=Kaleido)
+- Hyperledger Workshop: [Hyperledger FireFly and DIDs](https://www.youtube.com/watch?v=ICp6AX9zsW8&ab_channel=HyperledgerFoundation)
+- Webinar: [Launch Enterprise Scale NFT Experiences That Customer Love](https://www.youtube.com/watch?v=YrrSp2SKcfQ&ab_channel=Kaleido)
+- EthCC: [State of Enterprise Blockchain](https://www.youtube.com/watch?v=1XruoWm9cF4)
+- Point Zero Forum: [SME Financing via Programmable Money and Tokenization hosted by Bank of Thailand](https://www.pointzeroforum.com/insights/sme%20financing%20via%20programmable%20money%20and%20tokenisation?hsLang=en)
+
+Contributor activity updates:
+
+The star activity for the FireFly repository has continued to grow from 339 end of last quarter to 398 end of this quarter. Discord activity remains consistent with ~45 DAU.
+
+![alt_text](images/ffq3image1.png "image_tooltip")
```

Comment:
It might be worth while renaming this image to `ff2023q3image1.png`. Including the year will ensure no conflicts next year.

Reply:
Good suggestion. I was thinking from the directory structure that wouldn't be an issue since all the 2024 reports would be in a separate parent directory. But I can rename them if that's not the planned structure.

Reply:
My bad. I missed that the `images` folder was already in the `2023` folder. No change needed (unless I am too late)